cmd/resource.go

```diff
@@ -24,9 +24,156 @@ import (
 
 // resourceConfigCmd represents the resource subcommand
 var resourceConfigCmd = &cobra.Command{
-	Use:   "resource FLAG",
-	Short: "specific resources to be watched",
-	Long:  `specific resources to be watched`,
+	Use:   "resource",
+	Short: "manage resources to be watched",
+	Long: `
+manage resources to be watched`,
+	Run: func(cmd *cobra.Command, args []string) {
+		cmd.Help()
+	},
+}
+
+var resourceConfigAddCmd = &cobra.Command{
+	Use:   "add",
+	Short: "adds specific resources to be watched",
+	Long:  `adds specific resources to be watched`,
+	Run: func(cmd *cobra.Command, args []string) {
+		conf, err := config.New()
+		if err != nil {
+			logrus.Fatal(err)
+		}
+
+		var b bool
+		b, err = cmd.Flags().GetBool("svc")
+		if err == nil {
+			if b == true {
+				conf.Resource.Services = b
+				logrus.Info("resource svc configured")
+			}
+		} else {
+			logrus.Fatal("svc", err)
+		}
+
+		b, err = cmd.Flags().GetBool("deployments")
```

Comment:
`deployments` is plural while others are used as singular (e.g. `svc`), please
use singular, and let's use the same Kubernetes abbreviated form, i.e.
`deploy`.

Ditto below with other cases of this.

Reply:
changes done